### PR TITLE
migrate content from MSDN

### PR DESCRIPTION
**Pull request by v-kents**
## Summary

Migrate parameter information from MSDN

Fixes #70857 (https://ceapex.visualstudio.com/CPS/_workitems/edit/70857)

### CODE REVIEW

File: xml/System/WindowsRuntimeSystemExtensions.xml [hunk 1]
```diff
@@ -76,11 +99,24 @@
...
       <Docs>
-        <typeparam name="TResult">To be added.</typeparam>
-        <param name="source">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <typeparam name="TResult">The type that returns the result. </typeparam>
+        <param name="source">The started task. </param>
+        <summary>Returns a Windows Runtime asynchronous operation that represents a started task that returns a result. </summary>
+        <returns>A **Windows.Foundation.IAsyncOperation<TResult>** instance that represents the started task. </returns>
```
Comment (mairaw) [inline xml/System/WindowsRuntimeSystemExtensions.xml]: I imagine you can fix the build issue like this:

```suggestion
        <returns>A <see cref="T:Windows.Foundation.IAsyncOperation`1" /> instance that represents the started task. </returns>
```

File: xml/System/WindowsRuntimeSystemExtensions.xml [hunk 2]
```diff
@@ -76,11 +99,24 @@
...
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+> [!NOTE]
+> In Visual Basic and C#, you can call this method as an instance method on any object of type <xref:System.Threading.Tasks.Task>. When you use instance method syntax to call this method, omit the first parameter. For more information, see [Extension Methods (Visual Basic)](~/visual-basic/programming-guide/language-features/procedures/extension-methods.md) or [Extension Methods (C# Programming Guide)](~/csharp/programming-guide/classes-and-structs/extension-methods.md).
```
Comment (mairaw) [inline xml/System/WindowsRuntimeSystemExtensions.xml]:
```suggestion
> In Visual Basic and C#, you can call this method as an instance method on any object of type <xref:System.Threading.Tasks.Task>. When you use instance method syntax to call this method, omit the first parameter. For more information, see [Extension Methods (Visual Basic)](~/docs/visual-basic/programming-guide/language-features/procedures/extension-methods.md) or [Extension Methods (C# Programming Guide)](~/docs/csharp/programming-guide/classes-and-structs/extension-methods.md).
```

**Comment by mairaw** [comment]
Can we make the same changes as we did in the other PR to not change the CRLF?

**Comment by v-kents** [comment]
Dang, sorry. I had looked at it when I switched the other over, and it was saying LF then. Switched back.

**Comment by mairaw** [comment]
Thanks! I'm reviewing both PRs today extensively and hopefully we can merge both today!

**Pull request merged**